grid/app/main/events/dfl/syft_events.py

```diff
@@ -0,0 +1,51 @@
+import json
+from flask_login import current_user
```

Comment:
The grouping "standard python, external module, local" is being partly acknowledge here .